hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieColumnProjectionUtils.java

```diff
@@ -109,4 +109,25 @@ public static List<Pair<String,String>> getIOColumnNameAndTypes(Configuration co
         .collect(Collectors.toList());
   }
 
+  /**
+   * if schema contains timestamp columns, this method is used for compatibility when there is no timestamp fields
+   * We expect 3 cases to use parquet-avro reader to read timestamp column:
+   *  1. read columns contain timestamp type
+   *  2. no read columns and exists original columns contain timestamp type
+   *  3. no read columns and no original columns, but avro schema contains type
+   */
+  public static boolean supportTimestamp(Configuration conf) {
+    List<String> reads = Arrays.asList(getReadColumnNames(conf));
+    if (reads.isEmpty()) {
+      return getIOColumnTypes(conf).contains("timestamp");
+    }
+    List<String> names = getIOColumns(conf);
+    if (names.isEmpty()) {
+      return true;
```

Comment:
why return true？

Reply:
@xiarixiaoyao 
My consideration is to avoid that some hive related test cases do not explicitly declare these configurations, resulting in test failure. I have encountered some similar problems on azure pipeline, so I add it. If it is empty, it defaults to true.

Reply:
> My consideration is to avoid that some hive related test cases do not explicitly declare these configurations

@cdmikechen we should not change the code logic for the sake of fixing tests. If tests missing the config, then set it. Tests are to serve the main code, not the other way around.